Library/Formula/ufraw.rb

```diff
@@ -1,9 +1,8 @@
-require 'formula'
-
 class Ufraw < Formula
-  homepage 'http://ufraw.sourceforge.net'
-  url 'https://downloads.sourceforge.net/project/ufraw/ufraw/ufraw-0.20/ufraw-0.20.tar.gz'
-  sha1 'f2f456c6ec5ab128433502eae05b82a7ed636f3e'
+  homepage "http://ufraw.sourceforge.net"
+  desc "RAW image processing utility"
```

Comment:
The style so far has been `desc` at the top, above `homepage`. Let's keep it that way unless there's a formal decision made otherwise :).